### ISSUE TITLE
caseware - the job location does not correspond with the official website

### ISSUE THREAD

**Issue by AdelinaGuliman** [peviitor-ro/search-engine#438]
ENVIROMENT: Production
URL:https://peviitor.ro/rezultate?q=caseware&company=caseware&country=România&page=1
Browser: Safari
Device: Laptop
OS: MacOS

STEPS TO REPRODUCE:
1. Write in the search bar "caseware"
2. Click the button "Vezi postul ->" from the "SAP Developer" job

EXPECTED RESULTS:
The job location on [peviitor.ro](https://peviitor.ro/rezultate?q=caseware&company=caseware&country=România&page=1) corresponds to the job location on the [official website](https://jobs.lever.co/caseware/0676b59e-e420-46b1-9eca-4f9bf37b6957)

ACTUAL RESULTS:
The job location on [peviitor.ro](https://peviitor.ro/rezultate?q=caseware&company=caseware&country=România&page=1) does not correspond to the job location on the [official website](https://jobs.lever.co/caseware/0676b59e-e420-46b1-9eca-4f9bf37b6957)

https://github.com/peviitor-ro/search-engine/assets/137428958/7504e7fb-8fd0-4565-ad21-4cf6d919fdf1

**Comment by robert-burcea** [comment]
Yes but the front end feeds from the scraper, and that's what the scraper gets. Doesn't appear to be a front end issue.

![Screenshot_2024-02-06-17-15-35-27_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/peviitor-ro/search-engine/assets/28507505/20562fd4-648c-4b83-b536-009320877350)

**Comment by cristiol** [comment]
It s was a scraper related issue
fixed now
![image](https://github.com/peviitor-ro/search-engine/assets/142798921/d96d33dd-beb3-4194-8372-16819ff353ea)
